### ISSUE TITLE
Geodesic line drawing produced a line across the map 

### ISSUE THREAD

**Issue by MacaScull** [zakjan/mapbox-gl-draw-geodesic#6]
Hello, Im using this library so I can produce geodesic line and polygons on a map. Though when testing the linestring I noticed this issue. I was wondering if you were aware of this and have a plan to fix it any time soon?

Ive attached a picture and a gif of said issue below, a similar thing occurs when using polygons. 

![LinestringGeodesic](https://user-images.githubusercontent.com/35259402/117837104-10553400-b271-11eb-994c-cb1cf0f4e3c9.PNG)
![GeodesicLineGlitch](https://user-images.githubusercontent.com/35259402/117837114-13e8bb00-b271-11eb-88c9-28597515d5c3.gif)

**Comment by zakjan** [comment]
Hi, this is a common issue with all mapping libraries that allow repeating the world, because defining whether a long line should be drawn in eastwards or westwards direction is not standardized. What would be the expected result for your case?

**Comment by MacaScull** [comment]
Hi, thanks for the speedy reply. The coordinates I used were:
```
[[-82.2685688977058, 35.20429746474875],
[-65.27209029717348, -76.11022740266773],
[-215.05355796436305, -41.928433349134174]]
```
It only ever appears when crossing the 180th meridian east bound but when plotted there any additional plots on east of that point will produce the same issue though if you go west bound of the 180th meridian it wont appear. Ill upload a gif showing what I mean. 
![GeodesicLineGlitch2](https://user-images.githubusercontent.com/35259402/117947986-9ec8c480-b308-11eb-9a8a-bb85741f9453.gif)

**Comment by zakjan** [comment]
@MacaScull Thanks for the coordinates. It helped me to track down the bug in my calculations. Fixed in https://github.com/zakjan/mapbox-gl-draw-geodesic/commit/38023aecee57434d995920841ac6ef76f185a97a and released as 2.1.1. Could you confirm it works for you?

**Comment by zakjan** [comment]
Closing as fixed, please let me know otherwise.
